docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+
+*Vulkan Configurator* can be used using command line to setup the system environment. Use the commannd `vkconfig --help` for more information in this case.
+
+We recommand the using *Vulkan Configurator* GUI approach for Vulkan application developers, it's the most effectively approach to switch between multiple layers configurations and quickly iterate during the development process.
+Additionally, *Vulkan Configurator* presents to the Vulkan application developers the layers found on the system and the settings of each layers, allowing Vulkan application developers to discover new fonctionnalities from the GUI without having to dig into each layer documentation.
+
+### Enabling and ordering the layer with the override layer file (`VkLayer_override.json`)
+
+To control the enabled layers and the layer order, *Vulkan Configurator* is generating the `VkLayer_override.json` file which is consumed by the Vulkan loader to enable Vulkan layers and control the order of the layers. This file also store the user-defined paths specified in *Vulkan Configurator* to find additional layers.
+
+#### The override layer file on Linux and macOS
+
+Unix systems store override layer file in the following paths:
+
+- `$HOME/.local/share/vulkan/implicit_layer.d/VkLayer_override.json`
+
+#### The override layer file on Windows
+
+Windows systems store the override layer file in the following path:
+
+- `%HOME%\AppData\Local\LunarG\vkconfig\override\VkLayerOverride.json`
+
+### Configuring the layers using the settings file (`vk_layer_settings.txt`)
+
+To control the layer settings, *Vulkan Configurator* is generating the `vk_layer_settings.txt` file which is consumed by the Vulkan layers and set the setting values defined by the Vulkan developers using the UI.
+
+By default, the **Vulkan Layer Settings** library requires the settings file to be named `vk_layer_settings.txt` and it will search it in the working directory of the targeted application. Hence, if a file is found in the working directory of the targeted application, the **Vulkan Layer Settings** library will by pass the layer settings created by *Vulkan Configurator*.
+If `VK_LAYER_SETTINGS_PATH` is set and is a directory, then the settings file must be a file called `vk_layer_settings.txt` in the directory given by `VK_LAYER_SETTINGS_PATH`.
+If `VK_LAYER_SETTINGS_PATH` is set and is not a directory, then it must point to a file (with any name) which is the layer settings file.
+
+The settings file can be modified or fully hand-written by the Vulkan application developers or third party tools. The settings file consists of comment lines and settings lines. Comment lines begin with the `#` character. Settings lines have the following format:
+
+   `<`*`LayerName`*`>.<`*`setting_name`*`> = <`*`setting_value`*`>`
+
+The settings and values available for each layer are listed in the layer manifest and the documentation of each layer using **Vulkan Layer Settings** library to implement layer settings.
```

Comment:
Not sure about the meaning of the part about documentation, some clarification might be helpful.

Reply:
Clearily a problem happen here. \o/ Fixed